guides/v2.3/rest/modules/inventory/manage-sources.md

```diff
@@ -0,0 +1,246 @@
+---
+group: rest
+title: Manage sources
+---
+
+Sources represent locations storing and shipping available product stock. Any location with available stock and capable of order fulfillment can be added as a source. These locations can include warehouses, brick-and-mortar stores, distribution centers, and drop shippers.
+
+All stores begin with a default source that must remain enabled. Single Source merchants (merchants who ship all products from one location) use the default source for their single point of inventory location and shipments. Multi Source merchants create as many sources as they need to represent each location.
+
+You cannot rename, delete, or disable the default source. You can create, modify, enable, and disable custom sources, but you cannot rename or delete a custom source.
+
+Disabling a custom source has the following effects:
+
+* Magento ignores and does not list the source for shipment or order processing
+* Stocks do not access inventory quantities from the source for aggregated inventory totals
+* Order shipments cannot be assigned to disabled locations.
+
+{:.bs-callout .bs-callout-info}
+Bundle and grouped products currently do not support multi-sourcing and must be assigned to the default source and default stock.
```

Comment:
I believe grouped products are being supported with Milestone 2. Bundle is not supported yet. May need to 100% verify with Igor.

Reply:
I plan on merging this PR on Tuesday. If the code code PR that adds support for grouped product is merged at that time, I'll update this and the tutorial.